jenkins/bootstrap_test.py

```diff
@@ -71,9 +71,13 @@ class FakeSubprocess(object):
     """Keep track of calls."""
     def __init__(self):
         self.calls = []
+        self.tmp_contents = []
 
     def __call__(self, cmd, *a, **kw):
         self.calls.append((cmd, a, kw))
+        for arg in cmd:
+            if arg.startswith('/tmp/'):
```

Comment:
this seems a little fragile

Reply:
Fixed to just grab all absolute path args instead.